src/components/Layout/index.tsx

```diff
@@ -18,8 +22,54 @@ const navBarContent = (title: string, refUrl: string, isActive: boolean = false)
     </Link>
   );
 };
+  
+const { SUCCESS, ERROR } = ToastTypes;
+
+const SELF_URL = `${process.env.NEXT_PUBLIC_BASE_URL}/users/self`;
```

Comment:
I believe this is missed, you can shift `SELF_URL` to `constant` file and import it directly here.
The reason being we could reuse it else where as well